src/components/Notice/index.tsx

```diff
@@ -11,20 +10,19 @@ const Container = styled(Box)`
   display: flex;
   align-items: center;
 `
+
 type NoticeProps = {
   children: ReactNode
+  className?: string
 } & BoxProps
 
-const Notice = ({ children, ...props }: NoticeProps) => (
-  <Container {...props}>
+const Notice = ({ children, className, ...props }: NoticeProps) => (
+  <Container className={className} {...props}>
```

Comment:
any reason using for spread everything except className ?

Change not required it was just a question

Reply:
@chambo-e I let you answer haha

Reply:
@chambo-e I let you answer haha

Reply:
emotion require a type with className to be able to style them, the rest is xstyled props which we will remove soon :)